plugin/api.go

```diff
@@ -174,6 +174,9 @@ type API interface {
 	// Minimum server version: 5.6
 	SearchChannels(teamId string, term string) (*model.ChannelList, *model.AppError)
 
+	// SearchUsers returns a list of users based on some search criteria.
```

Comment:
#9616 intruded the documentation of the server version. Would you please document the required server version here?
```suggestion
	// SearchUsers returns a list of users based on some search criteria.
	//
	// Minimum server version: 5.6
```

Reply:
@hanzie updated .